sdk/typespec/typespec_client_core/examples/stream_response.rs

```diff
@@ -2,11 +2,19 @@
 // Licensed under the MIT License.
 
 use futures::StreamExt;
+use tracing::level_filters::LevelFilter;
+use tracing_subscriber::fmt::format::FmtSpan;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // Log traces to stdout.
+    let _log = tracing_subscriber::fmt()
+        .with_span_events(FmtSpan::ENTER | FmtSpan::EXIT)
+        .with_max_level(LevelFilter::DEBUG)
+        .init();
```

Comment:
Do we get sanitization for free? If not we should be careful to not dump access tokens

Reply:
We're dumping this exact source file, so no. The real tracing implementation will not dump PII.